### ISSUE TITLE
Remove dependencies on antd from the react cron widget

### ISSUE THREAD

**Issue by jdthorpe** [jdthorpe/ras-lights#14]
Currently the (otherwise completely awesome) react-js-cron widget depends on antd, which clobbers all my styles. There is an [open issue](https://github.com/xrutayisire/react-js-cron/issues/10) which promises to fix this issue.  Follow up when they fix this issue.

**Comment by jdthorpe** [comment]
Tentatively fixed (thought the buttons are kinda ugly...)

**Comment by slimandslam** [comment]
Is this code in a repo somewhere?

**Comment by jdthorpe** [comment]
@slimandslam my little hack is documented [here](https://github.com/xrutayisire/react-js-cron/issues/10#issuecomment-1060003207)

**Comment by slimandslam** [comment]
Yes. I saw that. It's been some months since they said they'd start working on making the widget UI-agnostic, but I don't see any progress anywhere. Oh well.

**Comment by jdthorpe** [comment]
Ya, that's how it goes with open source sometimes.  A lot of good intentions, without substantial motivation to follow through.  Hence my little good-enough-for-me hack...
